### DISCUSSION TITLE
Block unsigned images in some namespaces while detect this images without block in others

### DISCUSSION THREAD

**Discussion by albert0fg** [sse-secure-systems/connaisseur#691] [Ideas]
I think that it could be good to block unsigned images in a specific namespace and use detection mode on other namespaces.
When i use the namespace validation I can select ignore & validate, maybe  adding another tag in the validate mode, like "validate_warm"  could solve it.
Other option can be allowing detectionMode per namespace using tags.

Maybe it's possible to do it and I'm missing something...

If anyone can suggest anyway to do that, it will be appreciated

Thanks

**Comment by xopham** [comment]
@albertovmware that feature does not exist yet and `detectionMode` can only be configured globally. It may however make sense to allow overwriting the detection mode on a per namespace basis. I quite like the idea and would believe it should not be hard to do that. Feel free to open an actual issue on it :pray: 
Also feels rather k8s-native...

**Comment by JosephYostos** [comment]
Is this feature still doesn't exist ?
